airflow/auth/managers/base_auth_manager.py

```diff
@@ -457,14 +459,17 @@ def register_views(self) -> None:
         """Register views specific to the auth manager."""
 
     @staticmethod
-    def _get_token_signer():
+    def _get_token_signer(expiration_time_in_seconds: int = 0) -> JWTSigner:
         """
         Return the signer used to sign JWT token.
 
         :meta private:
         """
+        if expiration_time_in_seconds == 0:
+            expiration_time_in_seconds = conf.getint("api", "auth_jwt_expiration_time")
+
```

Comment:
```suggestion
```

Reply:
I updated all to get the conf value as default now, even for simple auth manager. Thanks for pointing! :)